external/vpx-idreamofjeannie/README.md

```diff
@@ -0,0 +1,34 @@
+# I Dream of Jeannie (Original 2019)
+
+![Table Preview](https://vpuniverse.com/screenshots/monthly_2019_03/552127063_IDreamofJeannie(Original2019)CryptDoctor21Xenonphfullscreen.PNG.2e1ff0135bfdaa0fbce950f3989e4496.PNG)
+
+Author: [cryptdoctor21](https://vpuniverse.com/profile/12711-cryptdoctor21/)  
+Version: 1.0.0  
+Download: [VP Universe](https://vpuniverse.com/files/file/5521-i-dream-of-jeannie-pinball-vpx/)
+
+DirectB2S
+
+Author: [RyDarquayle](https://www.vpforums.org/index.php?showuser=51255)  
+Version: 1.0 
+Download: [VP Universe] (DirectB2S file included in table zip)
+
+ROM
+
+Download: [VP Forums](https://pinballnirvana.com/forums/resources/genie.1863/)
+
+SHA: 86c91838e6f9564ee4b1421e7d24f8f3f908f5bf
+MD5: 02c3bb866e8c97642f61ea4e5c03ceb7
+
+## Status 
+
+Minimum VPX Standalone build: 10.8.0-1989-a764013
+| Playfield | Controls | Backglass | DMD | ROM Required | FPS | 
+|-----------|----------|-----------|-----|--------------|-----|
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | :white_check_mark: | 57 |
```

Comment:
No dmd?

Reply:
Table has no DMD. Scoring is kept on the backglass.